addons/social_share/views/snshare_frontend.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<odoo>
+    <template id="share_campaign_visitor" name="Share campaign">
```

Comment:
Why "visitor" naming ? It may conflict with visitor concept from website (we could think it has something to do with this model while not), so maybe ``share_campaign_frontend_main``simply ?